terraform/ecs-service/main.tf

```diff
@@ -191,10 +191,22 @@ resource "aws_kms_key_policy" "ecs_service_logs_key_policy" {
 
 resource "aws_cloudwatch_log_group" "ecs_service_logs" {
   # checkov:skip=CKV_AWS_338:Retains logs for 30 days instead of 1 year
-  name              = "/platform/${local.service_name}/ecs-service-logs"
+  name              = "/platform/ecs/service/${local.service_name}"
```

Comment:
See other comment, seems odd to loose the environment? If I look at all the logs I will see logs from every env and its more difficult to track it down. This also increases the complexity of parsing the env name for when we ship to sentinel. Unless I've missed something?